### PR TITLE
send previous server offer when reconnecting

### PR DESCRIPTION
**Pull request by cnderrauber**

### CODE REVIEW

File: src/proto/google/protobuf/timestamp.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
 /* eslint-disable */
-import Long from 'long';
```
Comment (cnderrauber) [inline src/proto/google/protobuf/timestamp.ts:2]: changed by protoc version? have already run `yarn install`, but still change these file

Comment (lukasIO) [reply src/proto/google/protobuf/timestamp.ts:2]: strange, this looks like prettier changes. can you run `yarn format` to see if that helps?

**Review by lukasIO — APPROVED** [review]

File: src/proto/google/protobuf/timestamp.ts (hunk 1 above)
Comment (cnderrauber) [reply src/proto/google/protobuf/timestamp.ts:2]: @lukasIO No change after `yarn format` ..

File: src/room/Room.ts [hunk 2]
```diff
@@ -1050,9 +1051,16 @@ class Room extends (EventEmitter as new () => TypedEmitter<RoomEventCallbacks>)
 
     this.engine.client.sendSyncState({
       answer: toProtoSessionDescription({
-        sdp: previousSdp.sdp,
-        type: previousSdp.type,
+        sdp: previousAnswer.sdp,
+        type: previousAnswer.type,
       }),
+      offer:
+        previousOffer !== null
```
Comment (davidzhao) [inline src/room/Room.ts:1058]: default value is undefined, which is diff from null. using "trueness" operator would be fine in this case.

```suggestion
        previousOffer ? toProtoSessionDescription
```

**Pull request merged**